### ISSUE TITLE
Reactivate code on /events when relevant API is documented

### ISSUE THREAD

**Issue by hollomancer** [OperationCode/front-end#402]
# Feature request
Uncomment code on events page and integrate with services when events API documentation is up.

**Comment by kylemh** [comment]
Blocked by back-end @wimo7083

**Comment by ksmacleod99** [comment]
@kylemh is there a specific issue this is blocked by?

**Comment by kylemh** [comment]
There's no back-end endpoints for events. We have the ability to make a UI based off of events on all our Meetups, but no way of easily getting that data.

**Comment by Lianis-Operations** [comment]
@kylemh would it be possible to integrate google calendar with the website?

**Comment by kylemh** [comment]
I'm not sure there is a backend for this.

**Comment by kylemh** [comment]
Certainly we could do that Lianis. Is that the only source we want to connect to?

**Comment by Lianis-Operations** [comment]
@kylemh I want to say yes since Slack is already connected. @hollomancer @datphotogurl what do you suggest?

**Comment by kylemh** [comment]
Gonna close this since we won't be making a UI regarding Meetup's API. Instead, we'll make a new ticket to create a UI outlining our Google Calendar.
